File: 2595. Number of Even and Odd Bits/solution.java
class Solution {
  public int[] evenOddBit(int n) {
    int even = 0;
    int odd = 0;
    for (int index = 0; index < 32; index++){
      if ((n & (1 << index)) != 0){
        if (index % 2 == 0){
          even += 1;
        }
        else{
          odd += 1;
        }
      }
    }
    return new int[] {even, odd};
  }
}

File: 2595. Number of Even and Odd Bits/solution.py
class Solution(object):
  def evenOddBit(self, n):
    """
    :type n: int
    :rtype: List[int]
    """
    even = 0
    odd = 0
    for index in range(32):
      if n & (1 << index):
        if index % 2 == 0:
          even += 1
        else:
          odd += 1
    return [even, odd]